kitsune/sumo/tasks.py

```diff
@@ -1,18 +1,15 @@
+import logging
 from datetime import datetime
 
 from celery import task
-from django_statsd.clients import statsd
 
-from kitsune.sumo.decorators import timeit
+
+log = logging.getLogger('k.task')
 
 
 @task()
-@timeit
 def measure_queue_lag(queued_time):
-    """A task that measures the time it was sitting in the queue.
-
-    It saves the data to graphite via statsd.
-    """
+    """A task that measures the time it was sitting in the queue."""
     lag = datetime.now() - queued_time
-    lag = (lag.days * 3600 * 24) + lag.seconds
-    statsd.gauge('celery.lag', max(lag, 0))
+    lag = max((lag.days * 3600 * 24) + lag.seconds, 0)
+    log.debug(f'Measuse queue lag task value is {lag}')
```

Comment:
typo

Reply:
Also, shouldn't we use `log.info` here so we see it appear in logs?

And we can just use `timedelta.total_seconds()` now instead of doing the maths ourselves: https://docs.python.org/3/library/datetime.html#datetime.timedelta.total_seconds